www/docs/guides/connect_network.md

```diff
@@ -35,10 +35,11 @@ On Starknet.js side, you have to select the proper version, to be in accordance
 | :---------------------------: | ---------------------------- |
 |            v0.4.0             | Starknet.js v5.21.1          |
 |            v0.5.0             | Starknet.js v5.23.0          |
-|            v0.5.1             | Starknet.js v5.29.0 & v6.0.0 |
-|            v0.6.0             | Starknet.js v6.0.0           |
+|            v0.5.1             | Starknet.js v5.29.0 & v6.1.5 |
+|            v0.6.0             | Starknet.js v6.2.0           |
+|            v0.7.0             | Starknet.js v6.2.0           |
 
-Starknet.js v6.x.x will recognize automatically if you are connected to a v0.5.1 or a v0.6.0 rpc spec version.
+[!NOTE] Each Starknet.js version 6.x.x is compatible with 2 rpc spec versions.
```

Comment:
I think it would be 3 now 0.5, 0.6, 0.7
Also it should recognize automatically spec version if not provided.

Reply:
Done.